src/autoschedulers/mullapudi2016/Makefile

```diff
@@ -15,6 +15,8 @@ endif
 
 CXXFLAGS += -I$(COMMON_DIR)
 
-$(BIN)/libautoschedule_mullapudi2016.$(SHARED_EXT): $(SRC)/AutoSchedule.cpp $(LIB_HALIDE)
```

Comment:
Should this be order-only instead? If `AutoSchedule.cpp` includes Halide.h, it needs a dependency on that somehow.

Reply:
...yeah, probably